package.json

```diff
@@ -14,15 +14,16 @@
   "main": "index.js",
   "scripts": {
     "test": "lab -cv",
-    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/stylish.js --exclude node_modules"
+    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/index.js --exclude node_modules"
```

Comment:
@phated Fair enough. Let's just sit on that for the time being, then. It's certainly not something you'd want to land without thorough scrutiny, I appreciate that.

I'm not really sure what to do with the eslint switch though. But if the PRs get forwarded automagically anyway, I suppose it hardly matters in which order things go down. I guess I'll just amend the PRs that are still open, and submit new ones for the other repos.